### PR TITLE
MV Extension API Updates

### PR DESCRIPTION
**Pull request by rahul-nagraj**

### CODE REVIEW

**Comment by rahul-nagraj** [comment]
#sign-off

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -7431,6 +7431,24 @@
...
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "db8cde72-c86c-4a67-9c0f-65663c90d0df",
+          "ApiChange": "Property",
+          "ChangedApiName": "isMultiValued",
+          "ChangeType": "Addition",
+          "Description": "Added the `isMultiValued` property to the [extensionProperty](https://learn.microsoft.com/en-us/graph/api/resources/extensionProperty?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:7441]: Suggestion to enhance the description to improve the discoverability of this capability. Dan has called out this capability for directory extensions and coming soon for schema extensions.
```suggestion
          "Description": "Added the `isMultiValued` property to the [extensionProperty](https://learn.microsoft.com/en-us/graph/api/resources/extensionProperty?view=graph-rest-beta) resource. You can now store a collection of values in a directory extension property.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @rahul-nagraj for your work on this PR. I've left feedback inline plus the following:
- Also, add the property to the properties table and JSON representation of [extensionproperty.md](https://learn.microsoft.com/en-us/graph/api/resources/extensionproperty?view=graph-rest-beta). And indicate that it supports $filter with eq.
- Add the property to the payloads in the LIST and GET examples.

NOTE: Let's sync internally so I can:
+ Update the comparison table in the [extensions overview article](https://learn.microsoft.com/en-us/graph/extensibility-overview?view=graph-rest-1.0&tabs=http#choose-an-extension-type-for-your-application) and document that directory extensions can be multi-valued.
+ Update the examples relating to directory extensions to show this property. It'll also be good to add an example of how to filter when the property is multivalued. Let's not allow this capability to go unnoticed.

**Comment by rahul-nagraj** [comment]
#feedback-addressed

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**